internal/hud/renderer.go

```diff
@@ -111,46 +113,57 @@ func (r *Renderer) layout(v view.View, vs view.ViewState) rty.Component {
 	split := rty.NewFlexLayout(rty.DirVert)
 
 	split.Add(r.renderResources(v, vs))
-	split.Add(r.renderStatusBar(v))
+	split.Add(r.renderFooter(v))
 	l.Add(split)
 
 	if vs.LogModal.TiltLog {
+		keys = "(esc) to exit view "
```

Comment:
1. should we instead be setting `keys` before we call `renderFooter`?
2. global mutable variables (`keys`) are generally not great, and leave us open to all sorts of issues. If we fix (1), then we can just make `keys` a local variable here and pass it to `renderFooter` as a parameter.